test/validations/singletonIsUnnamedIffIsLiteral.wlk

```diff
@@ -0,0 +1,16 @@
+// Top level objects must always be named
+object namedObject {
+
+  // Inline objects must always be unnamed
+  var f = object { }
+
+  method m() {
+    // Inline objects can't be unnamed
+    @Expect(code="singletonIsUnnamedIffIsLiteral", level="error")
+    object otherNamedObject { }
```

Comment:
```suggestion
    return object otherNamedObject { }
```

Reply:
@fdodino if we add the return, then the annotation would apply to the `Return` node instead of the object literal. If you want, we could write it:
```wollok
return @Expect(code="singletonIsUnnamedIffIsLiteral", level="error") object otherNamedObject { }
```
but I don't think the return is necessary, since Wollok's blocks returns the last line anyway.

Reply:
@nscarcella  At least for Wollok Xtext implementation it was necessary:

![image](https://user-images.githubusercontent.com/4549002/131228362-a3ddc936-16ce-4808-b0ba-cd1d2faa9b57.png)

See the difference using `return`:

![image](https://user-images.githubusercontent.com/4549002/131228383-a45513b6-f100-49ef-aff9-d84b16af0fe7.png)

So I just wanted to use the `return` keyword because I thought otherwise it could be confusing. But I don't want to put expectation after the return. For now lets keep it that way.

Reply:
@fdodino you are right. I was thinking about how blocks behave, not methods.

Anyway, there is no need to return anything in the method, so we can leave it as it is, but the expression there is a bit useless and I think some other validation might get triggered... We can of course wait and see.

Reply:
Yes, I'd like to avoid possible issues. So 

```wollok
f = object otherNamedObject { }
```

would be better for me than the current proposal.

Reply:
Sadly, that would mark the variable declaration node instead of the literal expression, so it's the same as the `return` case.

Reply:
```suggestion
  method m() =
    // Inline objects must be unnamed
    @Expect(code="singletonIsUnnamedIffIsLiteral", level="error")
    object otherNamedObject { }
```
?

Reply:
Qué te parece lo que propone @PalumboN, @fdodino ? Eso funcionaría...

Reply:
Sí, démosle para adelante...